package.json

```diff
@@ -76,15 +76,17 @@
     "@types/ws": "^6.0.2",
     "@types/yargs": "^13.0.2",
     "autoprefixer": "^9.6.1",
-    "bespoke": "^1.1.0",
+    "bespoke": "bespokejs/bespoke",
```

Comment:
We have to use Bespoke.js from GitHub directly because `Bespoke.prototype.destroy()` is not defined in the latest release on npm.